src/app/modules/item/components/user-progress/user-progress.component.html

```diff
@@ -14,33 +14,34 @@
       <i class="fa fa-check"></i>
     </span>
   </div>
-  <mat-menu
-    #menu="matMenu"
-    yPosition="above"
-    class="grid-td-menu"
-  >
-    <div class="content validated">
-      <span *ngIf="userProgress">
-        <i class="fa fa-clock"></i>
-        <span class="menu-label" i18n>Time spent</span>
-        <span class="menu-value">{{ userProgress.timeSpent | duration }}</span>
-      </span>
-      <span class="score">
-        <alg-score-ring
-          *ngIf="state !== 'success' && state !== 'not-started'"
-          [isDark]="true"
-          [currentScore]="userProgress.score"
-          [diameter]="32"
-        ></alg-score-ring>
-        <span class="situation-check" *ngIf="state === 'success'">
-          <i class="fa fa-check"></i>
-        </span>
-      </span>
-      <span *ngIf="canAccess" class="clickable" (click)="onAccess()">
-        <i class="fa fa-lock"></i>
-        <span i18n>ACCESS</span>
-      </span>
-    </div>
-    <span class="indicator"></span>
-  </mat-menu>
+
+  <p-overlayPanel #op appendTo="body" styleClass="user-progress-overlay-panel">
+    <ng-template pTemplate>
+      <div class="user-progress-info">
+        <div class="content validated">
+          <span *ngIf="userProgress">
+            <i class="fa fa-clock"></i>
+            <span class="menu-label" i18n>Time spent</span>
+            <span class="menu-value">{{ userProgress.timeSpent | duration }}</span>
+          </span>
+          <span class="score">
+            <alg-score-ring
+              *ngIf="state !== 'success' && state !== 'not-started'"
+              [isDark]="true"
+              [currentScore]="userProgress.score"
+              [diameter]="32"
+            ></alg-score-ring>
+            <span class="situation-check" *ngIf="state === 'success'">
+              <i class="fa fa-check"></i>
+            </span>
+          </span>
+          <span *ngIf="canAccess" class="clickable" (click)="onAccess()">
+            <i class="fa fa-lock"></i>
+            <span i18n>ACCESS</span>
```

Comment:
if I click access, the panel stay open together with the other panel.
![Screenshot 2021-06-09 at 11 34 51](https://user-images.githubusercontent.com/1053150/121330904-b64e9980-c916-11eb-8cfe-76dd376c6a63.png)

Reply:
Fixed